mmv1/products/networkconnectivity/InternalRanges.yaml

```diff
@@ -0,0 +1,151 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'InternalRange'
+base_url: 'projects/{{project}}/locations/global/internalRanges'
+create_url: 'projects/{{project}}/locations/global/internalRanges?internalRangeId={{name}}'
+self_link: 'projects/{{project}}/locations/global/internalRanges/{{name}}'
+update_verb: :PATCH
```

Comment:
We'll need at least one update test, see https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-an-update-test for instructions

Reply:
@rileykarson Added an update test, happy to receive your feedback!

Reply:
Something pops to mind, i might want to add another update test specifically for the `ipCidrRange` field to combine it with the `usage` field. I believe these are now not covered. How may i add another update test? I assume it's replicating the process?

Reply:
Yep, another TestAcc block and associated configs.

Reply:
@rileykarson Added another test, it should cover now all updatable fields.